apache-nifi.advisories.yaml

```diff
@@ -54,6 +62,10 @@ advisories:
             componentType: java-archive
             componentLocation: /usr/share/nifi/lib/nifi-hwx-schema-registry-nar-1.26.0.nar
             scanner: grype
+      - timestamp: 2024-05-17T08:31:36Z
+        type: pending-upstream-fix
+        data:
+          note: Upstream repo must mitigate this CVE
```

Comment:
Can we use the same updated note from previous review comment for all other instances?